### ISSUE TITLE
Cannot use text boxes under wayland

### ISSUE THREAD

**Issue by GiantNuker** [ppy/osu#14975]
Cannot type in text boxes (login, map search) under Wayland (swaywm), while game keys and menu keyboard navigation work. Text boxes work fine under Xorg (i3wm)

System: Arch Linux (everything is updated), SwayWM Wayland compositor
Osu Version: 2021.1005.0-lazer

Dosen't appear to be very useful but heres the [runtime.log](https://github.com/ppy/osu/files/7289831/runtime.log)
